app/bundles/ReportBundle/Builder/MauticReportBuilder.php

```diff
@@ -499,11 +499,13 @@ private function applyFilters(array $filters, QueryBuilder $queryBuilder, array
             $filterExpr = $groups[0];
         } elseif (count($groups) > 1) {
             // Sets of expressions grouped by OR
-            $orX = $queryBuilder->expr()->orX();
-            $orX->addMultiple($groups);
+            $orX = $queryBuilder->expr()->or('');
+            foreach ($groups as $group) {
+                $orX = $orX->with($group);
+            }
```

Comment:
Would this work?
```suggestion
            $orX = $orX->with(...$groups);
```

Reply:
I think this could work 

```$orX = $queryBuilder->expr()->or(...$groups);```

Reply:
@escopecz check this commit 813aaeecdfad20dea9f4240bd48eff425362fda2, maybe a good solution